client/src/components/ComboBoxPreview/ComboBoxPreview.scss

```diff
@@ -35,17 +38,20 @@
   overflow: hidden;
   display: flex;
   flex-direction: column;
+  // Use 65:35 ratio between the iframe and the details, but allow the details
+  // to grow to a minimum of 100px in case the container is very small
+  // (i.e. when there are only a few blocks)
+  min-height: max(35%, 100px);
 }
 
 .w-combobox-preview__label {
   @apply w-label-1;
-  max-height: 25%;
-  overflow: auto;
```

Comment:
I think making the label scrollable was too much. We don't expect the label to be very long as that'd be ugly for the combobox too anyway